bears/r/FormatRBear.py

```diff
@@ -1,22 +1,69 @@
 from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.parsing.StringProcessing import escape
 
 
+def _map_to_r_bool(py_bool):
+    return 'TRUE' if py_bool else 'FALSE'
+
+
 @linter(executable='Rscript',
         output_format='corrected',
         prerequisite_check_command=('Rscript', '-e', 'library(formatR)'),
         prerequisite_check_fail_message='Please install formatR for this bear '
                                         'to work.')
 class FormatRBear:
-
     """
     Check and correct formatting of R Code using known formatR utility.
     """
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
+        """
+        :param r_keep_comments:         Determines whether comments are
+                                        kept or not.
+        :param r_keep_blank_lines:      Determines whether blank lines
+                                        are kept or not.
+        :param r_braces_on_next_line:   Determines wheter a brace should
+                                        be placed on the next line
+                                        Example: if `True`,
+                                        "if (...)
+                                        {"   instead of "if (...) {"
```

Comment:
ahh I understand now, anyway this is not really well formatted.

1. missing period after "next line"
2. enclose the parameters inside backticks
3. wrap them a bit better
4. a space too much after first example
5. typo in "wheter"

Also I think you should make use of the new documentation style (see below), that makes stuff better readable^^

-->
```
:param r_braces_on_next_line:
    Determines whether a brace should be placed on the next line (instead of
    residing in the same one).

    If ``True``,
    ```
    if (...) {
    ```
    changes to
    ```
    if (...)
    {
    ```
```

And what happens if `False` is specified? Is brace alignment ignored then or is "brace-on-the-same-line" style enforced?